packages/cozy-konnector-libs/docs/api.md

```diff
@@ -452,11 +452,16 @@ You need the full permission on `io.cozy.files` in your manifest to use this fun
   + `validateFile` (function) default: do not validate if file is empty or has bad mime type
   + `validateFileContent` (boolean or function) default false. Also check the content of the file to
   recognize the mime type
+  + `filePrimaryKeys` (array of strings). Describe which attributes of files will be taken as primary key for
+  files to check if they already exist, even if they are moved. If not given, the file path will
+  used for deduplication as before.
```

Comment:
It looks like it is the same thing as `idAttributes`, would you consider renaming the variable ?

https://github.com/cozy/cozy-libs/blob/7a0a26b959af8cf7f9be0608c8e6b4648d927131/packages/cozy-doctypes/README.md#idattributes

Reply:
Good point, but then I would prefer fileIdAttributes since theses options are also given to saveBills (which uses the `keys` option for deduplication) and bills and files do not have the same primary key.

ex: a payment schedule file can be associated to 12 monthly bills

`saveBills` and then `hydrateAndFilter` could use an `idAttributes` alias for `keys` for consistency with `cozy-doctypes`

@edas what do you think

Reply:
Good catch @ptbrowne 
We should probably avoid creating different namings.

👍 with fileIdAttributes. `saveBills` indeed need a specific fileIdAttributes

`saveFile` could allow both idAttributes and fileIdAttributes so it keeps compatibility with both `saveBills` and the generic `idAttributes` naming.